modules/mobile/docs/openapi.yaml

```diff
@@ -536,6 +536,70 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments/facility/eligibility
+  /v0/appointments/va/eligibility:
+    get:
+      description: Lists types of service. For each type of service, lists users registered facilities that support request and direct appointments.
+      operationId: serviceEligibility
+      parameters:
+        - description: Array of facilities to be checked for service eligibility
+          in: path
```

Comment:
Should this be query instead? I think path means the parameter is part of the url, like /v0/mobile/facilities/:facility_id.

Reply:
yes, good catch